crates/libcontainer/src/user_ns.rs

```diff
@@ -78,6 +78,8 @@ pub enum UserNamespaceError {
     UnknownUnprivilegedUsernsClone(u8),
     #[error(transparent)]
     IDMapping(#[from] MappingError),
+    #[error(transparent)]
+    OtherIO(#[from] std::io::Error),
```

Comment:
Should the `OtherIO` be some other higher level error, as this is duplicated in another error enum below.